lib/node_modules/@stdlib/math/base/special/besselj1/docs/repl.txt

```diff
@@ -0,0 +1,30 @@
+
+{{alias}}( x )
+    Computes the Bessel function of the first kind of order one.
+
+    Parameters
+    ----------
+    x: number
+        Input value.
+
+    Returns
+    -------
+    y: number
+        Bessel function of the first kind of order one.
```

Comment:
What is returned is not so much the "Bessel function" but the function value. I would just replace with `Function value.`. (Aside: I have already updated `besselj0` similarly.)